src/main/java/eu/dissco/backend/service/SpecimenService.java

```diff
@@ -267,7 +254,7 @@ private Map<String, List<String>> mapParams(MultiValueMap<String, String> params
       throws UnknownParameterException {
     var mappedParams = new HashMap<String, List<String>>();
     for (var entry : params.entrySet()) {
-      var mappedParam = paramMapping.get(entry.getKey());
+      var mappedParam = MappingTerms.getParamMapping().get(entry.getKey());
```

Comment:
You can remove `MappingTerms` and just statically import this function

Reply:
Also, you can simplify this function. Instead of the function `mapParams` returning the whole hashmap, you could pass `entry.getKey()` as an argument and have` mapParams` just return the value.

Reply:
Also might be nice to make this an `Optional`